python/sdist/amici/petab/cli/import_petab.py

```diff
@@ -102,18 +115,42 @@ def _parse_cli_args():
     )
 
     args = parser.parse_args()
```

Comment:
don't we need a a `group.parse_args()` as well?

Reply:
No, that's happening automatically